test/unit/jsx.tsx

```diff
@@ -272,4 +272,275 @@ describe("snabbdom", function () {
       });
     });
   });
+
+  describe("Fragment", function () {
```

Comment:
The unit tests are good, but can you please add one or two browser tests to verify that the (flattened) children are also patched correctly?

Reply:
Hi @jvanbruegge,
While working on writing the test case, I noticed that a VNode like `{ sel: undefined, children: [...] }` is not a valid one while it typechecks.

I've submitted #975 that will treat such a VNode as a virtual `DocumentFragment`.
So could you please review that PR first?
Thanks!

Reply:
@jvanbruegge I've added a test case that ensures that virtual nodes made by JSX fragments can correctly be patched.
Please check it out!
https://github.com/snabbdom/snabbdom/pull/974/files#diff-2a05a44523d99a42362a290d489f088b0d55f4e74be980bb43ea5d1bc055e9b2R546-R598